app/helpers/codelist_helper.rb

```diff
@@ -12,8 +13,12 @@ def currency_select_options
     objects.unshift(OpenStruct.new(name: "Pound Sterling", code: "GBP")).uniq
   end
 
-  def benefitting_countries_hash
-    Codelist.new(type: "intended_beneficiaries").list
+  def benefitting_regions_for_form
+    BenefittingCountry::Region.all_for_level_code(BENEFITTING_SUB_REGION_2_CODE)
+  end
+
+  def benefitting_countries_in_region_for_form(region)
+    BenefittingCountry.non_graduated_for_region(region)
```

Comment:
Now this stuff has a degree of seperation from the codelist, whether it would be sensible to have a `BenefittingCountryHelper` or similarly named thing?

Reply:
We have a form helper - I should've started there! How does that sound?

Reply:
Yeah, that makes sense 👍